tests-functional/clients/services/wakuext.py

```
@@ -85,358 +86,356 @@ def __init__(self, client: RpcClient):
         super().__init__(client, "wakuext")
 
     def start_messenger(self):
-        response = self.rpc_request("startMessenger")
-        json_response = response.json()
-
-        if "error" in json_response:
-            assert json_response["error"]["code"] == -32000
-            assert json_response["error"]["message"] == "messenger already started"
-            return
+        try:
+            self.rpc_request("startMessenger")
+        except ApiResponseError as e:
+            if "messenger already started" in str(e):
```

Hmm, why would this ever be needed 🤔 
We should simply not try to start the messenger if it's started, and never silently ignore such errors. Looks like some old redundant workaround.

(not related to the PR, I just didn't see it before)

---

I've deleted it and tests (including reliability) still pass so guess we don't need it